docker-compose.yml

```diff
@@ -17,9 +17,7 @@ services:
       RABBIT_MQ_HOSTS: rabbitmq
       ELASTIC_SEARCH_HOSTS: elasticsearch
       METAX_DATABASE_HOST: db
-    configs:
-      - source: metax-web-config
-        target: '/code/metax_api/settings/.env'
+
```

Comment:
wasn't this working fine when you discovered that the configs could be updated dynamically?

Reply:
its just extra dependency, as the stack works without it just fine